liberty/src/utils/WindowOptions.hpp

```diff
@@ -0,0 +1,56 @@
+#pragma once
+#include <chrono>
+#include <functional>
+#include <concord/Computation.hpp> // for bolt::Metadata
+namespace concord {
+template <class ReducerType> struct WindowOptions {
+  public:
+  using ReducerFn =
```

Comment:
Do you think you would rather specify these as classes? 

Same deal, just easier to construct. no ? 

Just a thought.